### ISSUE TITLE
How do I read local manga collection?

### ISSUE THREAD

**Issue by TLiley2** [TachiWeb/TachiWeb-Server#48]
How do I read local manga collection?

**Comment by The7thSage** [comment]
I don't think you can right now, however Komga exists now. Komga is both a server and an extension for tachiyomi that scans your. I am just reinstalling Tachiweb to my laptop for the first time in months, so ill reply if the extension is even compatible,
Edit: nevermind me I dont think extensions work as planned for now.
